xyz.rescribe.rescribe.yml

```
@@ -0,0 +1,63 @@
+app-id: xyz.rescribe.rescribe
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: org.freedesktop.Sdk.Extension.golang
+build-options:
+  append-path: /usr/lib/sdk/golang/bin
+finish-args:
+  - --socket=x11
+  - --share=ipc # needed for X11
+  #- --socket=wayland # wayland is not stable on fyne yet, when it is add this and the wayland tag to 'go build'
+  - --device=dri # OpenGL
+  - --share=network # Used for google book downloading
+  - --filesystem=home
+command: rescribe
+modules:
+  - name: rescribe
+    buildsystem: simple
+    build-commands:
+      - cd cmd/rescribe && go build .
+      - install -Dm00755 cmd/rescribe/rescribe $FLATPAK_DEST/bin/rescribe-bin
+      - install -Dm00644 cmd/rescribe/icon.256.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/xyz.rescribe.rescribe.png
+      - install -Dm00644 cmd/rescribe/xyz.rescribe.rescribe.desktop $FLATPAK_DEST/share/applications/xyz.rescribe.rescribe.desktop
+      - install -Dm00644 cmd/rescribe/xyz.rescribe.rescribe.appdata.xml $FLATPAK_DEST/share/appdata/xyz.rescribe.rescribe.appdata.xml
+      - printf '#!/bin/sh\nTMPDIR=$XDG_RUNTIME_DIR rescribe-bin -gbookcmd "/app/bin/getgbook" -tesscmd "/app/bin/tesseract" -t "/app/share/tessdata/rescribev9_fast.traineddata"\n' > $FLATPAK_DEST/bin/rescribe
+      - chmod 755 $FLATPAK_DEST/bin/rescribe
+      - mkdir -p $FLATPAK_DEST/share/tessdata
+      - cp -r tessdata/* $FLATPAK_DEST/share/tessdata/
+    sources:
+      - type: git
+        url: https://github.com/rescribe/bookpipeline
+        tag: v1.0.2
+        commit: 3fb28b552eb1e945ff8ba48dd96271e641c295bc
+      - type: archive
+        url: https://rescribe.xyz/rescribe/modules-20221030-3a6024.tar.xz
+        sha256: 98fdbe455f7cb916cda570d0a30fca0de331809d4549e655e25b95e9be119856
+        strip-components: 0
+      - type: archive
+        url: https://rescribe.xyz/rescribe/embeds/tessdata.20220322.zip
+        sha256: 725fd570a3c3dc0eba9463248ce47a8646db8bafb198d428d6bb8f0be18540ee
+        strip-components: 0
+        dest: tessdata
+  - name: leptonica
+    sources:
+      - type: git
+        url: https://github.com/DanBloomberg/leptonica
+        tag: 1.82.0
+        commit: f4138265b390f1921b9891d6669674d3157887d8
+  - name: tesseract-ocr
+    sources:
+      - type: git
+        url: https://github.com/tesseract-ocr/tesseract
+        tag: 5.2.0
+        commit: 5ad5325a0aa8effc47ca033625b6a51682f82767
+  - name: getxbook
+    buildsystem: simple
+    build-commands:
+      - make DESTDIR=$FLATPAK_DEST PREFIX='' install
```

```suggestion
      - make PREFIX=$FLATPAK_DEST install
```

`DESTDIR` and `PREFIX` are two different things.

---

> `DESTDIR` and `PREFIX` are two different things.

Thanks for the suggestion, good call.

bot, build xyz.rescribe.rescribe